Vungle/VungleBannerCustomEvent.m

```diff
@@ -112,15 +112,7 @@ - (void)vungleAdDidLoad
     NSMutableDictionary *options = [NSMutableDictionary dictionary];
     
     if (self.localExtras != nil && [self.localExtras count] > 0) {
-        NSString *userId = [self.localExtras objectForKey:kVungleUserId];
```

Comment:
@YueVungle , @ashinagawa - just want to get confirmation here that the `user` play option is not valid for banners / MREC (I understand the update made on line 123 below where the wrong key was used to assign a value, but just want to be absolutely sure we should be removing the `user` play option. Thanks so much!